modules/op-installing-tkn-on-macos.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/tkn_cli/installing-tkn.adoc
+
+[id="installing-tkn-on-macos"]
+
+= Installing {op-title} on macOS
+
+== Binary installation
+
+. Download the binary :
++
+----
+# curl -L https://mirror.openshift.com/pub/openshift-v4/clients/tkn/latest/tkn-linux-amd64 -o /usr/local/bin/tkn
+----
++
+----
+# curl -L https://mirror.openshift.com/pub/openshift-v4/clients/tkn/latest/tkn-darwin-amd64 -o /usr/local/bin/tkn
```

Comment:
The binaries don't exist yet but @hrishin could confirm if the paths are correct.

Reply:
@siamaksade looks right